learn/safe-core/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -4,51 +4,104 @@ The [Auth kit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/a
+## Create SafeAuthKit Instance
+
+Create an instance of the [SafeAuthKit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/auth-kit/src/SafeAuthKit.ts) by providing the chosen adapter (e.g `Web3AuthAdapter`) and the kit configuration `SafeAuthConfig`.
 
 `Web3Auth` is the only provider type currently supported but we plan to add more providers in the future.
 
 ```typescript
-import { SafeAuthKit, SafeAuthProviderType } from '@safe-global/auth-kit';
+import { SafeAuthKit } from '@safe-global/auth-kit';
+import { Web3AuthOptions } from '@web3auth/modal';
+import { OpenloginAdapter } from '@web3auth/openlogin-adapter';
+
+// https://dashboard.web3auth.io/
+const WEB3_AUTH_CLIENT_ID=process.env.REACT_APP_WEB3_AUTH_CLIENT_ID!
 
 // https://web3auth.io/docs/sdk/web/modal/initialize#arguments
 const options: Web3AuthOptions = {
-  clientId: 'web3Auth-client-id',
+  clientId: WEB3_AUTH_CLIENT_ID,
   web3AuthNetwork: 'testnet',
-  chainConfig: { ... },
-};
+  chainConfig: {
+    chainNamespace: CHAIN_NAMESPACES.EIP155,
+    chainId: '0x5',
+    // https://chainlist.org/
+    rpcTarget: `https://rpc.ankr.com/eth_goerli`
+  },
+  uiConfig: {
+    theme: 'dark',
+    loginMethodsOrder: ['google', 'facebook']
+  }
+}
+
 // https://web3auth.io/docs/sdk/web/modal/initialize#configuring-adapters
-const modalConfig = { ... };
+const modalConfig = {
+  [WALLET_ADAPTERS.TORUS_EVM]: {
+    label: 'torus',
+    showOnModal: false
+  },
+  [WALLET_ADAPTERS.METAMASK]: {
+    label: 'metamask',
+    showOnDesktop: true,
+    showOnMobile: false
+  }
+}
+
 // https://web3auth.io/docs/sdk/web/modal/whitelabel#whitelabeling-while-modal-initialization
-const openloginAdapter = new OpenloginAdapter({ ... });
+const openloginAdapter = new OpenloginAdapter({
+  loginSettings: {
+    mfaLevel: 'mandatory'
+  },
+  adapterSettings: {
+    uxMode: 'popup',
+    whiteLabel: {
+      name: 'Safe'
+    }
+  }
+})
+
+const adapter = new Web3AuthAdapter(options, [openloginAdapter], modalConfig)
+
+const safeAuthKit = await SafeAuthKit.init(adapter, {
+  txServiceUrl: 'https://safe-transaction-goerli.safe.global'
+})
+```
 
-// Create an instance of the Web3AuthAdapter
-const web3AuthAdapter = new Web3AuthAdapter(options, [openloginAdapter], modalConfig);
+## Sign In to an Ethereum Account
 
-// Create an instance of the SafeAuthKit using the adapter and the SafeAuthConfig allowed options
-const safeAuthKit = await SafeAuthKit.init(web3AuthAdapter, { ... });
-```
+Once your Auth Kit instance is created, call `signIn()` to start the authentication process.
 
-Once the instance is created, you can call the `signIn()` method to start the authentication process showing the web3Auth modal in case you use the `Web3AuthAdapter`.
+When you sign in with the same social account, the same Ethereum address will be returned.
```

Comment:
When you sign in with the same social account, the same Ethereum address will be returned for the same client id.

Reply:
Thanks! Good catch! Important distinction.
```suggestion
When you sign in with the same social account, the same Ethereum address will be returned for the same Client ID.
```